WORKSPACE

```diff
@@ -137,12 +147,6 @@ overridden_targets = {
 
 load("@bazel_tools//tools/build_defs/repo:maven_rules.bzl", "maven_aar")
 
-# Because J2V8 is published as type `aar.asc`
-maven_aar(
```

Comment:
how does this part work now? i noticed you removed the dep for `android_binary`, I'm not following how _this_ source is fixed

Reply:
Somewhat hilariously, I think this is because it's a transitive dependency of `com.github.AlexTrotsenko:j2v8-debugger`, so it gets pulled into the `@maven` workspace.

I could be wrong, and maybe it just works now, but that's the main thing that changed.

Reply:
😅